CMakeLists.txt

```diff
@@ -528,7 +551,7 @@ if(MAKE_PYTHON_WRAPPER)
     endif()
   endif()
   
-  set_target_properties(${TARGET} PROPERTIES PREFIX "_" OUTPUT_NAME "gwb")
+  set_target_properties(${WB_TARGET} PROPERTIES PREFIX "_" OUTPUT_NAME "gwb")
```

Comment:
This change seem sot be tripping it up. Note that `${TARGET}` is not equal to `WorldBuilder` but to either `_gwb` or `gwb`. 

The main problem is that the c/cpp/fortran examples expect a library called libWorldBuilder.a, which with this change is no longer generated, but it instead generated `_gwb.a`.  Commenting out the line seems to let the testers pass on my system. I think this line may have been faulty an not executed.